### ISSUE TITLE
Spread operator breaking Node LTS

### ISSUE THREAD

**Issue by tquetano-r7** [google/closure-compiler-js#5]
There is an instance of the spread operator (and perhaps more) that causes this to break instantly in Node LTS (4.5.0). I'm sure this works just fine in the current version, but the lack of support for certain ES2015 features in the LTS prevents any use of this package.

**Comment by MatrixFrog** [comment]
The JS is created with GWT which I'm pretty sure only outputs ES5. Are you sure you're seeing a spread operator in the compiler?

**Comment by tquetano-r7** [comment]
There error I'm seeing is a node failure:

```
  out.warnings = [...out.warnings];
                  ^^^

SyntaxError: Unexpected token ...
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:373:25)
    at Object.Module._extensions..js (module.js:416:10)
```
And checking `compile.js` in the `node_modules` folder, it's definitely there.

```javascript
const jscomp = require('./jscomp.js');

module.exports = function(flags) {
  const clone = {};
  for (const k in flags) {
    clone[k] = flags[k];
  }
  const out = jscomp(clone);

  // hide weird GWT internals
  out.warnings = [...out.warnings];
  out.errors = [...out.errors];

  return out;
};
```

**Comment by tivac** [comment]
@MatrixFrog https://github.com/google/closure-compiler-js/blob/14052e4ad4c127ec531ec8fdabd420d8265d479f/compile.js#L34-L35

**Comment by MatrixFrog** [comment]
Oh, gotcha. It's not in the gwt-compiled JS, but the small amount of hand-written JS. @samthor what do you think about taking out the `...` until the LTS version of node supports it?

**Comment by samthor** [comment]
We can do that. I think there may be other reasons we don't support LTS though. I'm not really up to speed on what it does or doesn't support - I'm surprised it seems to support `const` and `let`?

@tquetano-r7 can you send a PR? Otw I'll get to it soon.

**Comment by samthor** [comment]
[Fixed here](https://github.com/google/closure-compiler-js/commit/bf97b9e5304985406276e3b6e476071bcd768943). We'll try to target the LTS release (currently 4.5) from now-on-in.

**Comment by planttheidea** [comment]
Sorry for the late response, appreciate you jumping on it so quickly!

**Comment by tquetano-r7** [comment]
Appreciate you guys fixing this, but is there any chance it can be released? Fresh install of the NPM package still points to 20160822.

**Comment by samthor** [comment]
I've published 20160822.0.1 which has the fix.
